packages/logger/src/keys.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/logger
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export namespace Logger {
+  export namespace SequenceActions {
+    export const SETUP_LOGGER  = 'logger.sequenceActions.setupLogger';
+  }
+  export namespace System {
+    export const LOGGER_PROVIDER  = 'logger.system.loggerProvider';
```

Comment:
1) IMO, the term "Provider" is redundant here. A provider is a way how to register a value provider with our IoC container (`Context`). The actual value injected won't be a provider, but the logger instance returned by the provider. In that sense, the key name may be even misleading.

Here is what I am proposing instead:

```ts
export const LOGGER = 'logger.system.logger';
```

2) Could you please elaborate little bit more on the "system" name - why should the binding key include "system" in the name? IMO, there is nothing system-wide or system-level in LoopBack Next, so it does not make sense to me to include "system" in the binding key. Especially when the binding is contributed by an optional component. It's perfectly possible that I am missing something, that's why I'd like to hear your thoughts.